config/dlss_baseline.yml

```diff
@@ -46,12 +46,6 @@ Metrics/MethodLength:
 Metrics/ModuleLength:
   Max: 200
 
-Style/IfUnlessModifier:
```

Comment:
Changes to this file were made to appease the latest Rubocop, which complains about outdated cops.